CHANGELOG.md

```diff
@@ -12,6 +13,9 @@ Add an entry to the unreleased provider section whenever merging a PR to main th
 * (deps!) [#1196](https://github.com/cosmos/interchain-security/pull/1196) Bump [ibc-go](https://github.com/cosmos/ibc-go) to [v7.2.0](https://github.com/cosmos/ibc-go/releases/tag/v7.2.0).
 * `[x/ccv/provider]` (fix) [#1076](https://github.com/cosmos/interchain-security/pull/1076) Add `InitTimeoutTimestamps` and `ExportedVscSendTimestamps` to exported genesis.
 
+### Cryptographic verification of equivocation
```

Comment:
The linked EPIC's task list also contains slashing issues, so it might make sense to put it in the title here as well?